app/scripts/modules/amazon/webpack.config.js

```diff
@@ -103,6 +103,14 @@ module.exports = {
         test: /\.css$/,
         use: [{ loader: 'style-loader' }, { loader: 'css-loader' }, { loader: 'postcss-loader' }],
       },
+      {
+        test: /\.svg$/,
+        issuer: {
+          test: /\.(tsx?|js)$/,
+        },
```

Comment:
we need this `issuer` block so that CSS/LESS files continue to use the same `file-loader` import — the thing SVGR does of adding a named `ReactComponent` export messes that up.